frontend/src/__tests__/cypress/cypress/support/commands/odh.ts

```diff
@@ -400,6 +400,18 @@ declare global {
           options: { path: { namespace: string; serviceName: string; recurringRunId: string } },
           response: OdhResponse<GoogleRpcStatusKF>,
         ) => Cypress.Chainable<null>) &
+        ((
+          type: `POST /api/service/pipelines/:namespace/:serviceName/apis/v2beta1/recurringruns/:recurringRunId:mode`,
+          options: {
+            path: {
+              namespace: string;
+              serviceName: string;
+              recurringRunId: string;
+              mode: string;
```

Comment:
could probably restrict this type more to `':enable' | ':disable'`, not gonna block the pr for it though